omod/src/main/resources/apps/overallActions_extension.json

```diff
@@ -29,5 +29,15 @@
         "icon": "icon-link",
         "order": 13,
         "requiredPrivilege": "Task: coreapps.mergeVisits"
+    },
+    {
+        "id": "${project.parent.groupId}.${project.parent.artifactId}.deletePatient",
+        "extensionPointId": "patientDashboard.overallActions",
+        "type": "script",
+        "label": "coreapps.task.deletePatient.label",
+        "script": "delPatient.showDeletePatientCreationDialog({{patient.uuid}})",
```

Comment:
yes but wluyima told me "you need to put quotes but as html entities
otherwise the library that evaluates the expression will escape the quotes
which makes the JS invalid"

On Thu, May 5, 2016 at 7:14 PM, Lluis Josep Martinez <
notifications@github.com> wrote:

> In omod/src/main/resources/apps/overallActions_extension.json
> <https://github.com/openmrs/openmrs-module-coreapps/pull/20#discussion_r62188257>
> :
>
> > @@ -29,5 +29,15 @@
> >          "icon": "icon-link",
> >          "order": 13,
> >          "requiredPrivilege": "Task: coreapps.mergeVisits"
> > +    },
> > +    {
> > +        "id": "${project.parent.groupId}.${project.parent.artifactId}.deletePatient",
> > +        "extensionPointId": "patientDashboard.overallActions",
> > +        "type": "script",
> > +        "label": "coreapps.task.deletePatient.label",
> > +        "script": "delPatient.showDeletePatientCreationDialog({{patient.uuid}})",
>
> Did you try *{\"{patient.uuid}\"}* ?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openmrs/openmrs-module-coreapps/pull/20/files/7ee16df360a259efbd44696db4fe3cdcb2b030aa#r62188257>
>